service/controller/clusterapi/v27/key/cluster.go

```diff
@@ -292,6 +311,38 @@ func baseRoleARN(cluster v1alpha1.Cluster, accountID string, kind string) string
 	return fmt.Sprintf("arn:%s:iam::%s:role/%s-%s-%s", partition, accountID, clusterID, kind, EC2RoleK8s)
 }
 
+func ensureLabel(labels string, key string, value string) string {
```

Comment:
Also taken from `kubernetesd`.

Reply:
Would be good to have unit test for this.

Reply:
This code is not new. It works. It never changes. I do not want to spent more time on this now.